From bdd9281dd964c36ac94ef3a9fd5d4f7dd24a3d06 Mon Sep 17 00:00:00 2001
From: Martin Olsson <martin@minimum.se>
Date: Mon, 26 Mar 2012 10:40:45 +0200
Subject: [PATCH] Fix compile error "isatty was not declared"; missing
 unistd.h include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

When I built apitrace on Fedora 16, using the usual:

	cmake -H. -Bbuild
	make -C build

...then I got this compile error:

	apitrace/cli/cli_dump.cpp: In function ‘int command(int, char**)’:
	apitrace/cli/cli_dump.cpp:176:25: error: ‘isatty’ was not declared in this scope
---
 cli/cli_dump.cpp |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/cli/cli_dump.cpp b/cli/cli_dump.cpp
index 28c91af..f52b83c 100644
--- a/cli/cli_dump.cpp
+++ b/cli/cli_dump.cpp
@@ -28,6 +28,9 @@
 #include <string.h>
 #include <limits.h> // for CHAR_MAX
 #include <getopt.h>
+#ifndef _WIN32
+#include <unistd.h> // for isatty()
+#endif
 
 #include "cli.hpp"
 #include "cli_pager.hpp"
-- 
1.7.8.6

